### DISCUSSION TITLE
mleader setting "text_right_attachment_type"

### DISCUSSION THREAD

**Discussion by pvk988** [mozman/ezdxf#674] [Q&A]
Hello.
How to set mleader setting "text_right_attachment_type" = bottom of top text line & underline top text line?
i use AutoCad.

ml_builder = msp.add_multileader_mtext("EZDXF", dxfattribs={'text_right_attachment_type': 6, 'text_left_attachment_type': 6})  # dxfattribs={'text_right_attachment_type': 7, 'text_left_attachment_type': 7}
ml_builder.quick_leader("Line1", target=Vec2(40, -15), segment1=Vec2.from_deg_angle(45, 14))

does not give a result

**Comment by mozman** [comment]
The `MULTILEADER` support is not officially released, this is also the only reason why `ezdxf` v0.18 isn't released yet. The `MULTILEADER` entity is neither fully tested nor documented, so please be patient.

BTW: the way you create the  `MULTILEADER` entity is not idiomatic, see the [multileader.py](https://github.com/mozman/ezdxf/blob/master/examples/entities/multileader.py) example (`dxfattribs` do not work in this context)

**Reply by pvk988** [reply]
Thank you!
I would like to be able to use the mleade style already configured in the file. Then to create mleade it is enough to specify: style mleade, 2 points ,text ,general scale mleade
